src/hooks/useBasisForm/validation/validateAutoComplete.ts

```diff
@@ -0,0 +1,14 @@
+import { AutoCompleteProps } from "../../../components/AutoComplete/types";
+
+export const validateAutoComplete = <Item>(
+  value: Item | null,
+  autoCompleteProps: AutoCompleteProps<Item>
+) => {
+  const { optional } = autoCompleteProps;
+
+  if (value === null) {
+    return optional ? null : "Required";
+  }
```

Comment:
```suggestion
  if (value === null && !optional) {
    return "Required";
  }
```

and let the return at the end handle the null.